### ISSUE TITLE
Cannot find module 'ts-toolbelt' or its corresponding type declarations

### ISSUE THREAD

**Issue by bmacher** [RangerMauve/mqtt-pattern#7]
When using `mqtt-pattern` and trying to compile the code I get the following error: 

```sh
$ tsc --noEmit
node_modules/mqtt-pattern/index.d.ts:2:24 - error TS2307: Cannot find module 'ts-toolbelt' or its corresponding type declarations.

2 import type { F } from "ts-toolbelt";
                         ~~~~~~~~~~~~~


Found 1 error in node_modules/mqtt-pattern/index.d.ts:2

error Command failed with exit code 2.
```

Problem is that `ts-toolbelt` is neither a `dependency` nor a `peerDependency` so it will not be installed with this library. My current workaround is to install it myself as `devDependency`. However, it would be better if `mqtt-pattern` would handle this.

Spec:
mqtt-pattern: 2.0.0
node: v18.12.1
typescript: 4.9.3

**Comment by RangerMauve** [comment]
Oh jeeze, @erichardson-lee set up the TS bindings and I'm honestly not sure how it works.

If you have PR that would fix this I'd be down to review and merge. :)

**Comment by erichardson-lee** [comment]
Oopsie, I missed rhat in testing.

I'm away right now (until the 15th) however, a temporary fix would be to install ts-toolbelt.
I'll make a MR when I'm home to address this. 
(Or if someone else wants to, look at adding it as a peer dependency/normal dependency)

**Comment by bmacher** [comment]
I can give it a try, if you want. :-)

**Comment by bmacher** [comment]
> If you have PR that would fix this I'd be down to review and merge. :)
@RangerMauve I created a PR. See #8 :)

**Comment by RangerMauve** [comment]
Should be fixed as of 2.0.1 Thanks @bmacher !
